IdentityCore/src/cache/accessor/MSIDDefaultTokenCacheAccessor.m

```diff
@@ -87,8 +90,13 @@ - (BOOL)saveTokensWithConfiguration:(MSIDConfiguration *)configuration
     result = [self saveIDTokenWithConfiguration:configuration response:response context:context error:error];
 
     if (!result) return result;
-
-    // Save SSO state (refresh token and account)
+    
+    //Save App metadata
+    result = [self saveAppMetadataWithConfiguration:configuration response:(MSIDAADTokenResponse *)response context:context error:error];
+    
+    if (!result) return result;
+    
+    // Save SSO state (refresh token and account and metadata)
```

Comment:
nit: this one doesn't save metadata anymore so the comment needs to be updated

Reply:
done